src/caselawclient/Client.py

```diff
@@ -694,6 +694,20 @@ def copy_judgment(self, old, new):
             accept_header="application/xml",
         )
 
+    def break_checkout(self, judgment_uri):
+        uri = self._format_uri(judgment_uri)
+        xquery_path = os.path.join(ROOT_DIR, "xquery", "break_judgment_checkout.xqy")
+        vars = json.dumps(
+            {
+                "uri": uri,
```

Comment:
we may want to come back and allow 'deep' to be set in Python, but no need to add it now.